KasianenkoTymur/HW7/task7_3.py

```diff
@@ -0,0 +1,12 @@
+def calculates_characters(string):
```

Comment:
good job
look at this implementation
def count_symbols_in_row(word):
    
    result = {}
    
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result